cvat-ui/src/components/profile-page/security-content/api-token-form.tsx

```diff
@@ -0,0 +1,132 @@
+// Copyright (C) CVAT.ai Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import React from 'react';
+import dayjs from 'dayjs';
+
+import Form from 'antd/lib/form';
+import Input from 'antd/lib/input';
+import DatePicker from 'antd/lib/date-picker';
+import Checkbox from 'antd/lib/checkbox';
+import Button from 'antd/lib/button';
+import { Row, Col } from 'antd/lib/grid';
+import Typography from 'antd/lib/typography';
+
+import { ApiTokenModifiableFields, ApiToken } from 'cvat-core-wrapper';
+
+interface Props {
+    onSubmit: (data: ApiTokenModifiableFields) => void;
+    onCancel: () => void;
+    submitting: boolean;
+    token: ApiToken | null;
+    tokenCount: number;
+}
+
+interface FormData {
+    name: string;
+    expirationDate: dayjs.Dayjs | null;
+    isReadOnly: boolean;
+}
+
+function ApiTokenForm({
+    onSubmit, onCancel, submitting, token, tokenCount,
+}: Props): JSX.Element {
+    const [form] = Form.useForm<FormData>();
+    const isEditing = !!token;
+
+    const handleSubmit = async (): Promise<void> => {
+        try {
+            const values = await form.validateFields();
+            onSubmit({
+                name: values.name,
+                expiryDate: values.expirationDate ? values.expirationDate.toISOString() : null,
+                readOnly: values.isReadOnly,
+            });
+        } catch (error) {
+            // Form validation failed
+        }
+    };
+
+    const getInitialExpirationDate = (): dayjs.Dayjs | null => {
+        if (isEditing) {
+            return token?.expiryDate ? dayjs.utc(token.expiryDate).local() : null;
+        }
+        return dayjs().add(1, 'year');
```

Comment:
Both returns UTC time in my case:

console.log(dayjs())
console.log(dayjs.utc())

2025-10-02T05:50:50.410Z
2025-10-02T05:50:50.411Z

The same about .toISOString() method. 

My local time is +3 hours 8:50

However 
console.log(dayjs().isUTC()) // False
console.log(dayjs.utc().isUTC()) // True

Could you please investigate about timezone?